cmd/manager/main.go

```diff
@@ -64,6 +65,10 @@ func main() {
 		klog.Fatal(err)
 	}
 
```

Comment:
can we clarify in the commit or in a code comment why this is needed now?

Reply:
e.g Add configv1 to scheme so the machine controller can fetch the Infrastructure object to set the clusterID

Reply:
done, I updated commit description